### PR TITLE
Implemented New Designs

### PR DESCRIPTION
**Pull request by TomSesselmann**
Updated all screens and added styling to the vote slider.

### CODE REVIEW

**Comment by TomSesselmann** [comment]
@XertroV you don't need to go through all of it line by line, just see if there's anything obvious that you would do differently.

File: web/src/Components/Btn.elm [hunk 1]
```diff
@@ -35,9 +36,13 @@ btn props inner =
                 Small ->
                     [ class "btn-outer--small" ]
 
+                VSmall ->
+                    [ vary BtnSmall True ]
+
                 Click msg ->
                     [ onClick msg ]
 
+                -- TODO: Btn is not safely disabled. Can be removed with Dev Tools
```
Comment (XertroV) [inline web/src/Components/Btn.elm:45]: That's fine - should have a check for appropriate conditions somewhere. Remember that a sophisticated user will just figure out how to issue the request manually if they want to.

Security and UX are two diff considerations, and we shouldn't handle security when thinking about UX stuff - that should be way deeper.

File: web/src/Components/IssueCard.elm [hunk 2]
```diff
@@ -22,119 +24,118 @@ issueCard model ballotId =
...
 
-        clickMsg =
-            if ballotDone then
-                Nav <| NTo <| ResultsR ballotId
-            else
-                Nav <| NTo <| VoteR ballotId
-
-        cardColor =
+        isPending =
+            ballot.state
```
Comment (XertroV) [inline web/src/Components/IssueCard.elm]: this is super unclear and takes a lot of mental energy to parse - should be changed to testing elements, like `List.member ballot.state [BallotPendingCreation, BalltoPendingEdits, BallotPendingDelegation] || vote.state == Vote.Pending`

That's just one example - other ways to do it but this isn't good.

File: web/src/Components/IssueCard.elm [hunk 3]
```diff
@@ -22,119 +24,118 @@ issueCard model ballotId =
...
-                displayResults
-            else if ballot.start > model.now then
-                "This ballot is not open yet"
-            else if checkAlreadyVoted ballotId model then
-                "You have voted in this ballot"
-            else
-                "You have not voted in this ballot yet"
+                IssueVoteNow
+
+        struct =
```
Comment (XertroV) [inline web/src/Components/IssueCard.elm]: if any of the following 20 lines is common it should be generated programatically - like there's a LOT of similar code here. Maybe use a make function mapping over a list of tuples or something.

File: web/src/Components/Slider.elm [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+                ]
+                empty
+
+        htmlSlider =
+            el Slider [ width fill, verticalCenter ] <|
+                html <|
+                    H.input
+                        (sliderOptions
+                            ++ [ HA.type_ "range"
+                               , HA.min <| toString <| range.min
```
Comment (XertroV) [inline web/src/Components/Slider.elm:48]: Should these be strings?

File: web/src/Components/IssueCard.elm (hunk 3 above)
Comment (XertroV) [reply web/src/Components/IssueCard.elm]: This comment is just to make the above part of the review

File: web/src/Components/IssueCard.elm (hunk 2 above)
Comment (XertroV) [reply web/src/Components/IssueCard.elm]: this comment is just to make the above part of the review

File: web/src/Components/Slider.elm [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+sliderLabel : Bool -> Float -> SvElement
+sliderLabel isDisabled value =
+    let
+        struct =
+            case value of
+                (-3) ->
+                    { l = 0, r = 1, txt = "Strongly Disagree" }
+
+                (-2) ->
+                    { l = 2, r = 13, txt = "Mostly Disagree" }
```
Comment (XertroV) [inline web/src/Components/Slider.elm:88]: Why are these numbers here? they seem arbitrary...

**Review by XertroV — CHANGES_REQUESTED** [review]

**Pull request merged**